.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,20 @@
+name: "Rclone config"
+
+query-filters:
+  - exclude:
```

Comment:
@ncw You can either globally surpress warnings here or locally supress them with a special comment.